pyrolite/util/classification.py

```diff
@@ -400,6 +400,46 @@ def __init__(self, **kwargs):
         super().__init__(**poly_config)
 
 
```

Comment:
If you add `@update_docstring_references` here it should clean up a few references in this submodule; I'll look at adding it above also to clean up the docs generation process (this makes sure for each docs page there aren't duplicated references to different things, which it doesn't handle well).

Reply:
Do you mean as a decorator to the class? It is already on the function.

Reply:
Ah right! Might actually be OK, I'll check the docs after merging.